README.md

```diff
@@ -1,3 +1,22 @@
 # python-tutorial
 
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/empa-scientific-it/python-tutorial.git/main?labpath=index.ipynb)
\ No newline at end of file
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/empa-scientific-it/python-tutorial.git/main?labpath=index.ipynb)
+
+
+## Run the tutorial on your computer
+
+To run the tutorial locally, setup the environment with [conda](https://docs.conda.io/en/latest/miniconda.html) (or [mamba](https://mamba.readthedocs.io/en/latest/installation.html)):
+
+```console
+conda env create -f environment.yml
+```
+
+Then activate the environment with
+```console
+conda activate aiidalab-docker-stack
```

Comment:
Shouldn't the environment name be `python-tutorial`? Or am I missing something?

Reply:
Yep, correct. Sorry for that 😄

Reply:
fixed in [28b16a8](https://github.com/empa-scientific-it/python-tutorial/pull/23/commits/28b16a8b8366dc15c7772074c4e2bcbf22f03ff6)

Reply:
Great!